### ISSUE TITLE
Create SRA Crawler

### ISSUE THREAD

**Issue by flaneuse** [NIAID-Data-Ecosystem/nde-crawlers#50]
- Access [SRA](https://www.ncbi.nlm.nih.gov/sra/) via [API](https://www.ncbi.nlm.nih.gov/books/NBK25501/). 
- Store each accession number as an individual dataset. 
- Note: will need to make sure to grab just the metadata for a [record](https://www.ncbi.nlm.nih.gov/sra/SRX14673411[accn]), not the actual sequencing data.

NOTE: Will need to consider how we also incorporate related NCBI projects: [BioProject](https://www.ncbi.nlm.nih.gov/bioproject) and [GenBank](https://www.ncbi.nlm.nih.gov/genbank/).

Basically -- also see more on the divisions [here](https://www.ncbi.nlm.nih.gov/guide/dna-rna/): 
- SRA = raw sequencing data
- GenBank = processed sequencing data to create whole genomes
- BioProject = metadata about the experiment that was used to create the SRA and/or GenBank data, including BioSamples, etc.
